Marlin/src/HAL/HAL_STM32/watchdog_STM32.cpp

```diff
@@ -20,13 +20,13 @@
  *
  */
 
-#ifdef STM32F7xx
+#if defined STM32F0xx || defined STM32F1xx || defined STM32F4xx || defined STM32F7xx
```

Comment:
By convention the `defined` directive should always include parentheses.

Reply:
Maybe "ARDUINO_ARCH_STM32" would bet better to use? See comment above.